src/main/java/com/cloudbees/jenkins/support/impl/RootCAs.java

```diff
@@ -129,8 +129,8 @@ public static void getRootCAList(StringWriter writer) {
       // Inspired by:
       // https://github.com/jenkinsci/jenkins-scripts/pull/82/files
       // https://stackoverflow.com/questions/8884831/listing-certificates-in-jvm-trust-store
-      final TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
-      trustManagerFactory.init((KeyStore) null);
+      final TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm()); //FIPS OK: Reading certificates.
+      trustManagerFactory.init((KeyStore) null); //FIPS OK: Not Used.
       TrustManager[] trustManagers = trustManagerFactory.getTrustManagers();
```

Comment:
I would add a reference to the documentation, like:
```suggestion
/*
FIPS OK: https://downloads.bouncycastle.org/fips-java/BC-FJA-UserGuide-1.0.2.pdf Section 7 Key Stores says:
The FIPS keystore type will read both BCFKS files and JKS files with one caveat, it will not accept a JKS file containing a secret/private key. This can be useful where JKS files are being used solely for the storage of certificates, such as with the cacerts file found in the typical JVM.
Meaning, reading certificates from a JKS file is fine.
*/
      TrustManager[] trustManagers = trustManagerFactory.getTrustManagers();
```